cluster/gce/config-test.sh

```diff
@@ -316,12 +316,17 @@ if [[ -n "${GCE_GLBC_IMAGE:-}" ]]; then
 fi
 
 if [[ -z "${KUBE_ADMISSION_CONTROL:-}" ]]; then
-  ADMISSION_CONTROL="MutatingAdmissionWebhook,Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority"
+  ADMISSION_CONTROL="Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,NodeRestriction,Priority"
   if [[ "${ENABLE_POD_SECURITY_POLICY:-}" == "true" ]]; then
     ADMISSION_CONTROL="${ADMISSION_CONTROL},PodSecurityPolicy"
   fi
   # ResourceQuota must come last, or a creation is recorded, but the pod may be forbidden.
-  ADMISSION_CONTROL="${ADMISSION_CONTROL},ValidatingAdmissionWebhook,ResourceQuota"
+  #
+  # MutatingAdmissionWebhook is a mutator, it's called in a phase before
+  # ResourceQuota, which is a validator. Hence, although
+  # MutatingAdmissionWebhook appears after ResourceQuota in this list, it's
+  # actually called before ResourceQuota at runtime.
+  ADMISSION_CONTROL="${ADMISSION_CONTROL},ValidatingAdmissionWebhook,ResourceQuota,MutatingAdmissionWebhook"
```

Comment:
We'll still run them in the right order, but I might place `MutatingAdmissionWebhook` before `ValidatingAdmissionWebhook` for appearances sake.

Reply:
I'm fine changing this.

Although having a sensible order here will lull people into a false sense of security ;)